Gemfile

```diff
@@ -73,9 +73,10 @@ group :test do
   gem "selenium-webdriver"
 end
 
-gem "phlex-rails"
+gem "phlex", github: "phlex-ruby/phlex"
+gem "phlex-rails", github: "phlex-ruby/phlex-rails"
 
-gem "ruby_ui", github: "ruby-ui/ruby_ui", branch: "main"
+gem "ruby_ui", github: "stephannv/ruby_ui", branch: "require_phlex_2"
```

Comment:
@stephannv , you can now point to ruby_ui again.

Reply:
I opened this PR https://github.com/ruby-ui/ruby_ui/pull/186 to fix some stimulus problems before pointing to main again.

Reply:
I had to adapt some code to work with RubyUI, but I tried to minimize the changes.